diff -rNEZbwBdu1 a/cygwin/core_portme.mak b/cygwin/core_portme.mak
--- a/cygwin/core_portme.mak	2020-05-28 01:02:54.000000000 +0800
+++ b/cygwin/core_portme.mak	2020-08-10 17:15:57.674992578 +0800
@@ -26,3 +26,3 @@
 #	Use this flag to define compiler options. Note, you can add compiler options from the command line using XCFLAGS="other flags"
-PORT_CFLAGS = -O2
+PORT_CFLAGS = -Ofast
 FLAGS_STR = "$(PORT_CFLAGS) $(XCFLAGS) $(XLFLAGS) $(LFLAGS_END)"
diff -rNEZbwBdu1 a/freebsd/core_portme.mak b/freebsd/core_portme.mak
--- a/freebsd/core_portme.mak	2020-05-28 01:02:54.000000000 +0800
+++ b/freebsd/core_portme.mak	2020-08-10 17:16:33.963284041 +0800
@@ -26,3 +26,3 @@
 #	Use this flag to define compiler options. Note, you can add compiler options from the command line using XCFLAGS="other flags"
-PORT_CFLAGS = -O2
+PORT_CFLAGS = -Ofast
 FLAGS_STR = "$(PORT_CFLAGS) $(XCFLAGS) $(XLFLAGS) $(LFLAGS_END)"
diff -rNEZbwBdu1 a/linux/core_portme.mak b/linux/core_portme.mak
--- a/linux/core_portme.mak	2020-05-28 01:02:54.000000000 +0800
+++ b/linux/core_portme.mak	2020-08-10 17:16:51.951428564 +0800
@@ -26,3 +26,3 @@
 #	Use this flag to define compiler options. Note, you can add compiler options from the command line using XCFLAGS="other flags"
-PORT_CFLAGS = -O2
+PORT_CFLAGS = -Ofast
 FLAGS_STR = "$(PORT_CFLAGS) $(XCFLAGS) $(XLFLAGS) $(LFLAGS_END)"
diff -rNEZbwBdu1 a/linux64/core_portme.mak b/linux64/core_portme.mak
--- a/linux64/core_portme.mak	2020-05-28 01:02:54.000000000 +0800
+++ b/linux64/core_portme.mak	2020-08-10 17:17:04.707531069 +0800
@@ -26,3 +26,3 @@
 #	Use this flag to define compiler options. Note, you can add compiler options from the command line using XCFLAGS="other flags"
-PORT_CFLAGS = -O2
+PORT_CFLAGS = -Ofast
 FLAGS_STR = "$(PORT_CFLAGS) $(XCFLAGS) $(XLFLAGS) $(LFLAGS_END)"
diff -rNEZbwBdu1 a/macos/core_portme.mak b/macos/core_portme.mak
--- a/macos/core_portme.mak	2020-05-28 01:02:54.000000000 +0800
+++ b/macos/core_portme.mak	2020-08-10 17:17:19.907653231 +0800
@@ -26,3 +26,3 @@
 #	Use this flag to define compiler options. Note, you can add compiler options from the command line using XCFLAGS="other flags"
-PORT_CFLAGS = -O2
+PORT_CFLAGS = -Ofast
 FLAGS_STR = "$(PORT_CFLAGS) $(XCFLAGS) $(XLFLAGS) $(LFLAGS_END)"
diff -rNEZbwBdu1 a/simple/core_portme.mak b/simple/core_portme.mak
--- a/simple/core_portme.mak	2020-05-28 01:02:54.000000000 +0800
+++ b/simple/core_portme.mak	2020-08-10 17:17:32.299752841 +0800
@@ -26,3 +26,3 @@
 #	Use this flag to define compiler options. Note, you can add compiler options from the command line using XCFLAGS="other flags"
-PORT_CFLAGS = -O2
+PORT_CFLAGS = -Ofast
 FLAGS_STR = "$(PORT_CFLAGS) $(XCFLAGS) $(XLFLAGS) $(LFLAGS_END)"
